bower.json

```diff
@@ -1,19 +1,19 @@
 {
-    "name": "purescript-arraybuffer-types",
-    "license": [
-        "MIT"
-    ],
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/purescript-contrib/purescript-arraybuffer-types"
-    },
-    "ignore": [
-        "**/.*",
-        "node_modules",
-        "bower_components",
-        "output"
-    ],
-    "dependencies": {
-        "purescript-psci-support": "^v5.0.0"
-    }
+  "name": "purescript-arraybuffer-types",
+  "license": [
+    "MIT"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/purescript-contrib/purescript-arraybuffer-types"
+  },
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "dependencies": {
+    "purescript-psci-support": "master"
+  }
```

Comment:
This should be a `devDependency`. And with that, this isn't even a breaking change!